### ISSUE TITLE
Why is the Windows Installer only available within a zip file?

### ISSUE THREAD

**Issue by HerrNamenlos123** [chalet-org/chalet#335]
Hi there, I am just starting out with Chalet and I am very interested in the project. I think it could become something amazing and thus the minimum I can do to help is to report inconveniences in order to help improve the tool.

I just wanted to download the tool but all downloads I can find, on Github and the Website, are all .zip files. I pretty much hate downloading zip files. Not because I can't unzip them but the moment I saw the .zip links, it felt like a hobby project to me and not like a polished tool. I then unzipped the file and the installer .msi is right in there. Why can't the download be just the .msi so that I can click on it immediately?

So, is there a specific reason why it is in a zip file? If not, would there be any interest to change this? In my opinion this is a bad first impression and might even prevent people from even installing the tool, because seeing only .zips is like "Oh, this only seems to be a hobby project and not suitable for me"... Changing it could improve first-time experience by quite a bit.

**Comment by rewrking** [comment]
I agree on all points, but it's not a super high priority - the installer zip is actually the most-downloaded file type. The technical reason is because I couldn't upload an exe with `softprops/action-gh-release@v1` for some reason. It wouldn't upload a .deb file either, and I haven't bothered to look for better options. The CI build process has to be redone anyway -- I only just made some kind of test suite that I can use to check for regressions.

I'd actually prefer an MSI for the windows installer, but I went down the Nullsoft installer route years ago and figured that would be good enough, At the time, you couldn't even generate a Visual Studio project.
